keyboards/hackpad/info.json

```
@@ -0,0 +1,77 @@
+{
+    "manufacturer": "Nico Stuhlmueller",
+    "keyboard_name": "hackpad",
+    "maintainer": "ThePurox",
+    "bootloader": "caterina",
```

```suggestion
    "development_board": "promicro",
```
Setting `"development_board": "promicro"` automatically sets `processor`, `bootloader` and `pin_compatible` appropriately; the `pin_compatible` setting is needed to make [converters](https://docs.qmk.fm/#/feature_converters) work with your board.